src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -552,6 +558,12 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     context.system.scheduler.scheduleOnce(ResendInterval, self, message)
   }
 
+  private def removePendingCallKeyPersistence(data: WorkflowData, message: PersistenceMessage) = {
+    logger.debug(s"Got whenUnhandled message: $message")
```

Comment:
This line should go in the message handling block rather than this private function